requirements.txt

```diff
@@ -3,5 +3,5 @@ numpy
 lsst-daf-butler @ git+https://github.com/lsst/daf_butler@main
 lsst-utils @ git+https://github.com/lsst/utils@main
 lsst-resources @ git+https://github.com/lsst/resources@main
-lsst-pipe-base @ git+https://github.com/lsst/pipe_base@main
-lsst-ctrl-mpexec @ git+https://github.com/lsst/ctrl_mpexec@main
+lsst-pipe-base @ git+https://github.com/lsst/pipe_base@tickets/DM-41711
+lsst-ctrl-mpexec @ git+https://github.com/lsst/ctrl_mpexec@tickets/DM-41711
```

Comment:
You generally want the changes to this file to be in their own commit and not mixed with other code changes so that you can drop the commit on rebase before merging. That's usually easier than editing it and then trying to found the commit to squash into.